.github/workflows/create-static-assets-on-release.yml

```diff
@@ -0,0 +1,36 @@
+name: Create static assets
+on:
+  release:
+    types: [published, edited]
+jobs:
+  static-assets:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout branch @${{github.ref_name}}
+        uses: actions/checkout@v3
+        with:
+          ref: ${{github.ref_name}}
+
+      - name: Setup python
+        uses: actions/setup-python@v4
+        with:
+          python-version: '3.9'
+
+      - name: Download requirements
+        run: |
+          pip install -r requirements.txt
+          cp .env.example .env
+
+      - name: Run collectstatic
+        run: python manage.py collectstatic --noinput
```

Comment:
Before running `collectstatic` you need to build the frontend Svelte code in order to include the frontend static files.. Please take a look at the [Docker build steps for building the frontend code](https://github.com/centerofci/mathesar/blob/c714892f3a0529cd65fadca8440b6cc3cb489ec0/Dockerfile#L66)